script/predicate_functions.sh

```diff
@@ -65,14 +65,6 @@ function is_ruby_23_plus {
   fi
 }
 
-function rspec_rails_compatible {
```

Comment:
I think this needs to be kept, 1.8.7 won't work with edge `rspec-rails` due to the new hash syntax:
```
gem 'yard', '~> 0.8.7', require: false

                                ^. Bundler cannot continue.
```

Reply:
Yeah this isn't present in dev and it should be... sorting...